examples/spark-viz/build.sbt

```diff
@@ -46,7 +46,7 @@ val ScalaCompatibleVersion = "2.12"
 // Change the dependency scope to "provided" when you run "sbt assembly"
 val dependencyScope = "compile"
 
-val geotoolsVersion = "1.4.0-28.2"
```

Comment:
Please remove the change of this file as it is not related to the binder notebooks.